README.md

```diff
@@ -2,8 +2,9 @@
 > Welcome to my portfolio!
```

Comment:
- [ ] Kindly add a description to your readme so that anyone visiting your repo has an idea of what this project is about.